ebook/1_latex2html.py

```diff
@@ -0,0 +1,737 @@
+# by Torben Menke https://entorb.net
+
+"""
+run from within ebook dir via 
+python3 1_latex2html.py
+"""
+
+import os
+import re
+import glob
+from datetime import date
+
+# Notes
+# footnotes are converted to inline text
+# these intro parts are skipped:
+#  Based on the characters of J. K. ROWLING
+#  CONTENT WARNINGS
+# Omake chapters are included inplace, not at appendix
+
+# TODO:
+# modify Preamble
+
+today = date.today()
+
+dir_tex_source = "../chapters/"
+dir_tmp = "tmp"
+dir_out = "output"
+for dir in (dir_tmp, dir_out):
+    os.makedirs(dir, exist_ok=True)
+
+
+html_preamble = f"""
+<h2>Preamble of this eBook</h2>
+<p>The original version of this great book 'Harry Potter and the Methods of Rationality' by Eliezer Yudkowsky is:<br/>
+<a href="https://www.hpmor.com">https://www.hpmor.com</a></p>
+<p>The typesetting as well as the reviewed text are adopted from:<br/>
+<a href="https://github.com/rjl20/hpmor">https://github.com/rjl20/hpmor</a></p>
+<p>Source code of converter script can be found at:<br/>
+<a href="https://github.com/rjl20/hpmor/ebook/">https://github.com/rjl20/hpmor/ebook/</a></p>
+<p>This eBook was created {today}</p>
+"""
+# TODO
+# add download location of .epub files in new repo
+# feel free to modify further
+
+# <p>This book is not my work, I just converted the text into eBook formats.</p>
```

Comment:
"into eBook formats" → "into ePUB format"

Reply:
actually, I am generating .epub as well as .mobi and hence wrote eBook formats. ok for  you to keep it as it is?

Reply:
In that case, I think normally it would be written something like "e-book formats"; "eBook" to me looks like a typo/thinko for "ePUB". A bit pedantic, I know!

Reply:
ok changed, you are the native speaker ;-)

Reply:
all changes are performed in an overwritten version of the initial commit of the branch "ebook".